ebs/ebs.go

```diff
@@ -589,6 +598,15 @@ func (d *Driver) CreateVolume(req Request) error {
 		if err := fs.FormatDevice(volume.Device, d.DefaultFSType); err != nil {
 			return err
 		}
+		log.Debugf("Launching snapshot creation for brand new volume: %s", volumeName)
+		request := &CreateSnapshotRequest{
```

Comment:
Would it make sense to consolidate this with the code around line 393 into a snapshot-launch function to keep it DRY?